src/N2WFacade.php

```diff
@@ -33,6 +34,8 @@ class N2WFacade extends Facade
      */
     protected static function getFacadeAccessor(): Transformer
     {
+        static::checkConfigIsPublished();
```

Comment:
Nếu như người dùng không publish file config thì file config mặc định của package sẽ được sử dụng bạn kham khảo tại [đây](https://github.com/phpviet/laravel-number-to-words/blob/master/src/ServiceProvider.php#L42).

Reply:
Mình vẫn không rõ tại sao nhưng có vẻ `mergeConfigFrom()` không hoạt động